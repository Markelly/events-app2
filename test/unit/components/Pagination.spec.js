import { shallowMount } from '@vue/test-utils'
import Pagination from '@/components/Pagination'

describe('<Pagination />', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(Pagination)
  })

  test('should render the component', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
