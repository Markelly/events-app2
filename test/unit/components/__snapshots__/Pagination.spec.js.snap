// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pagination /> should render the component 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
